src/pages/home/report/ReportActionsView.js

```diff
@@ -33,7 +33,7 @@ const propTypes = {
     report: reportPropTypes.isRequired,
```

Comment:
Just noticed, this file has permissions being changed

Reply:
This PR also proposes to change file permissions. I think you'll need to reset it.

<img width="1491" alt="Screenshot 2022-12-19 at 11 00 15 PM" src="https://user-images.githubusercontent.com/32012005/208485129-0831b8c9-3721-4210-8129-1f89c75e32e6.png">

Reply:
Wow, never seen this, never would notice 😂

Reply:
Wow, I've also never seen that! Thanks for noticing.